pkg/cmd/cli/cmd/env.go

```diff
@@ -260,7 +265,19 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 				return nil
 			}
 			for _, c := range containers {
+				oldEnv := c.Env
+				for _, env := range remove {
+					_, ok := findEnv(c.Env, env)
+					if !ok {
+						infosMsgs[i] = append(infosMsgs[i], fmt.Sprintf("environment variable %q not found in %s container\n", env, c.Name))
```

Comment:
nah, if I ask to remove envvar B, and it exists in one container but not another, I don't consider that failure or warning-worthy